tests/suites/test_suite_block_cipher.psa.data

```diff
@@ -0,0 +1,39 @@
+# These tests behave differently depending on the presence of
+# drivers and/or built-in, so they're isolated here for the benefit of
+# analyze_outcomes.py (driver vs reference comparison).
```

Comment:
Yet I'm not seeing `block_cipher.psa` in the list of ignored suites for this pair of components. I guess that's because in the reference component, `CIPHER_C` is enabled, and nothing's accelerated, so `BLOCK_CIPHER_C` is disabled, so the whole `test_suite_block_cipher` is skipped in the ref component, hence doesn't need an ignore entry.

Perhaps just remove the end of the sentence then (after the coma)?